src/common/components/clusterConfiguration/NetworkConfigurationFormFields.tsx

```diff
@@ -9,22 +9,16 @@ type NetworkConfigurationFormFieldsProps = NetworkConfigurationProps & {
   infraEnv?: InfraEnv;
 };
 
-const NetworkConfigurationFormFields: React.FC<NetworkConfigurationFormFieldsProps> = ({
-  ...props
-}) => {
-  const { cluster } = props;
-  return (
-    <Form>
-      <NetworkConfiguration {...props} />
-      <TextContent>
-        <Text component="h2">Security</Text>
-      </TextContent>
-      <ClusterSshKeyFields
-        clusterSshKey={cluster.sshPublicKey}
-        imageSshKey={props.infraEnv?.sshAuthorizedKey}
-      />
-    </Form>
-  );
-};
-
+const NetworkConfigurationFormFields: React.FC<NetworkConfigurationFormFieldsProps> = (props) => (
+  <Form>
+    <NetworkConfiguration {...props} />
+    <TextContent>
+      <Text component="h2">Security</Text>
+    </TextContent>
+    <ClusterSshKeyFields
+      clusterSshKey={props.cluster.sshPublicKey}
+      imageSshKey={props.infraEnv?.sshAuthorizedKey}
+    />
```

Comment:
This is duplicated in ClusterDeploymentNetworkingForm. How about renaming ClusterSshKeyFields to SecurityFields and include the title?